app/models/concerns/reminders/states/scheduled.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Reminders::States::Scheduled < Reminders::States::State
+  def initialize; end;
+
+  def notify reminder
+    reminder.issue.users.each do |user|
+      ReminderMailer.reminder_issue(reminder, user).deliver_later
+    end
+
+    reminder.update state_class_type: 'done'
```

Comment:
```suggestion
    reminder.update! state_class_type: 'done'
```
Y por lo mismo que en el anterior, perdón por no haberlo visto en las otras revisiones.

Reply:
olvidate yo tambien me lo comi jajajaja